packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 - Zowe Explorer now includes support for the [VS Code display languages](https://code.visualstudio.com/docs/getstarted/locales) French, German, Japanese, Portuguese, and Spanish. [#3239](https://github.com/zowe/zowe-explorer-vscode/pull/3239)
 - Localization of strings within the webviews. [#2983](https://github.com/zowe/zowe-explorer-vscode/issues/2983)
+- Leverage the new error correlation facility to provide user-friendly summaries of API and network errors. Extenders can also contribute to the correlator to provide human-readable translations of error messages, as well as tips and additional resources for how to resolve the error. [#3243](https://github.com/zowe/zowe-explorer-vscode/pull/3243)
```

Comment:
Technically anyone using the Zowe Explorer API can leverage the error correlator. This was per Ana's suggestion to make changelog entries more of an "advertisement/announcement" for what devs can do rather than technical details of the change.